skycoin/dmsg
# Monitor and log high CPU/mem consumption

WIP on https://github.com/SkycoinPro/skywire-services/issues/266

 Changes:	
- Monitor and log high CPU/mem consumption

How to test this PR:
- Run the integration env
- Check dmsg discovery or server logs
